python/smqtk/web/search_app/modules/file_upload/FileUploadMod.py

```diff
@@ -9,6 +9,7 @@
 
 from smqtk.utils import SmqtkObject
 from smqtk.utils import file_utils
+from six import BytesIO
```

Comment:
This is a duplicate import. Please remove.

Reply:
Removed